LandSandBoat/server
# Add piece of shiromochi

**_I affirm:_**

## What does this pull request do?

Adds piece of shiromochi and the +1 variant to the usable item list and adds the correct stats for the food.

## Steps to test these changes

Eat the food, verify that the stats for it are correct with /checkparam <me> as well as your main stats.  Use antacid or let it wear off to make sure the buffs are removed.
